src/utils.jl

```diff
@@ -38,23 +38,85 @@ function _get_label_plot(gp::GraphPlot, labels)
     end
 end
 
+"""
+    getedgeinds(gr::G, edgedat::D) where {G<:AbstractGraph, K<:AbstractEdge, D<:AbstractDict{K}, IsDirected{G}}
+
+Return enumeration of edges for directed graph
+"""
+@traitfn function getedgeinds(gr::G, edgedat::D) where {G<:AbstractGraph, K<:AbstractEdge, D<:AbstractDict{K}; IsDirected{G}}
```

Comment:
i think `getedgekeys` would be more fitting since indices are allways Integers?

Reply:
ok